src/SFML/Graphics/ImageLoader.cpp

```diff
@@ -272,6 +281,44 @@ bool ImageLoader::saveImageToFile(const std::string& filename, const std::vector
     return false;
 }
 
+////////////////////////////////////////////////////////////
+bool ImageLoader::saveImageToMemory(const std::string& format, std::vector<sf::Uint8>& output, const std::vector<Uint8>& pixels, const Vector2u& size)
+{
+    // Make sure the image is not empty
+    if (!pixels.empty() && (size.x > 0) && (size.y > 0))
+    {
+        // Choose function based on format
```

Comment:
Similarly to `loadFromFile`, and since the function is already there, you should use `format = toLower(format)` so that "PNG" is as valid as "png".

Reply:
Fixed.